ValgrindCI/parse.py

```diff
@@ -118,6 +118,8 @@ class ValgrindData:
     def __init__(self) -> None:
         self.errors: List[Error] = []
         self._source_dir: Optional[str] = None
+        self._substitute_paths: Optional[List[dict]] = []
+        self._relative_prefixes: Optional[List[str]] = []
```

Comment:
Why are these members declared `Optional[]` ? It doesn't seem to me they can ever be `None`...
```suggestion
        self._substitute_paths: List[dict] = []
        self._relative_prefixes: List[str] = []
```